Add new fields in user client for CTE users. (#17370)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/callrecords-clientuseragent.md

|headerValue|String|User-agent header value reported by this endpoint.|
|platform|microsoft.graph.callRecords.clientPlatform|Identifies the platform used by this endpoint. Possible values are: `unknown`, `windows`, `macOS`, `iOS`, `android`, `web`, `ipPhone`, `roomSystem`, `surfaceHub`, `holoLens`, `unknownFutureValue`.|
|productFamily|microsoft.graph.callRecords.productFamily|Identifies the family of application software used by this endpoint. Possible values are: `unknown`, `teams`, `skypeForBusiness`, `lync`, `unknownFutureValue`, `azureCommunicationServices`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `azureCommunicationServices`.|

> |communicationServiceId|String|Immutable resource identifier of the Azure Communication Service associated with User agents based on Communication Services APIs.|

Reviewer: For consistency with the other descriptions here, consider changing this from "associated with User agents" to "associated with this endpoint"
```suggestion
|communicationServiceId|String|Immutable resource identifier of the Azure Communication Service associated with this endpoint based on Communication Services APIs.|
```

> |azureADAppId|String|The unique identifier for the application that is assigned to an application by Azure AD.|

Reviewer: ```suggestion
|azureADAppId|String|The unique identifier of the Azure AD application used by this endpoint.|
```
